conf/machine/imx8ulp-lpddr4-evk.conf

```diff
@@ -0,0 +1,28 @@
+#@TYPE: Machine
+#@NAME: i.MX 8ULP EVK
+#@SOC: i.MX8ULP
+#@DESCRIPTION: Machine configuration for NXP i.MX 8ULP Evaluation Kit with LPDDR4
+#@MAINTAINER: Jun Zhu <junzhu@nxp.com>
+
+require include/imx8ulp-evk.inc
+
+KERNEL_DEVICETREE_BASENAME = "imx8ulp-evk"
+
+KERNEL_DEVICETREE += " \
+    freescale/imx8ulp-evk-epdc.dtb \
+    freescale/imx8ulp-evk-flexio-i2c.dtb \
+    freescale/imx8ulp-evk-i3c.dtb \
+    freescale/imx8ulp-evk-lpa.dtb \
+    freescale/imx8ulp-evk-lpspi-slave.dtb \
+    freescale/imx8ulp-evk-nd.dtb \
+    freescale/imx8ulp-evk-rk055hdmipi4m.dtb \
+    freescale/imx8ulp-evk-rk055hdmipi4mv2.dtb \
+    freescale/imx8ulp-evk-sof-btsco.dtb \
```

Comment:
Use `${KERNEL_DEVICETREE_BASENAME}` here.

Reply:
Thanks, @angolini, but there are two more parts, using `append:use-nxp-bsp`, and moving to `.inc`. Actually, leaving it here is fine for now. We can move anything common to `.inc` if/when we add `ddr4` version.